### ISSUE TITLE
why arm_loss = 0 and odm_loss = 0?

### ISSUE THREAD

**Issue by whcjb** [sfzhang15/RefineDet#57]
Hi, Zhang,
When I fine-tune the voc&coco model on the widerface datasets, it some time comes out arm_loss and odm_loss = 0, the whole loss is not zero.
    Iteration 1030, loss = 8.67766
    I0515 18:38:45.768995 29596 solver.cpp:259]     Train net output #0: arm_loss = 0 (* 1 = 0 loss)
    I0515 18:38:45.769006 29596 solver.cpp:259]     Train net output #1: odm_loss = 0 (* 1 = 0 loss)
Have you have seen this before, and what makes this?
Thanks!

**Comment by zhangxixi0904** [comment]
Hi. I got the same problem when trainning on Caltech dataset.

**Comment by sfzhang15** [comment]
@whcjb @zhangxixi0904 
Since we have not got this problem, we do not know the exact reason. We guess that is it because there is no positive sample？

**Comment by sfzhang15** [comment]
＠zhangxixi0904
1. Yes, the gts will be resized automatically.
2. You just need modify the number class to 2, the code will be run successfully. But if you want to get better results, you should modify the anchor (aspect ratio, scale, layer), data augmentation and other parameters, e.g., IoU threshold to divide positives and  negatives in ARM and ODM.
